pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -87,10 +79,18 @@ func ApplyMonitoringConsole(client splcommon.ControllerClient, cr splcommon.Meta
 		if err != nil {
 			return err
 		}
-		multiSite = clusterInfo.MultiSite
+		multiSite := clusterInfo.MultiSite
+		if multiSite == "true" {
```

Comment:
(if multSite) vs (if multiSite == "true")?

Reply:
"multSite" here is a string, not a bool

Reply:
Ah ok my bad!